internal/testpeertls/revocations.go

```diff
@@ -62,6 +100,6 @@ func NewRevokedLeafChain() ([]crypto.PrivateKey, []*x509.Certificate, pkix.Exten
 		return nil, nil, pkix.Extension{}, err
 	}
 
-	newChain, revocation, err := RevokeLeaf(keys, certs)
+	newChain, revocation, err := RevokeLeaf(keys[0], certs)
```

Comment:
is it possible for keys to be nil or have a length of 0 ?

Reply:
no